src/Artsy/Analytics/trackExperimentViewed.tsx

```diff
@@ -0,0 +1,20 @@
+import * as Schema from "Artsy/Analytics"
+import { data as sd } from "sharify"
+
+declare const window: any
```

Comment:
This `tracking` prop can be added to the typings in [global.d.ts](https://github.com/artsy/reaction/blob/5d2229bf76e8a0620a67a7b655d9d4f65ad060d1/src/Typings/global.d.ts#L15-L18).